test/completion_info.test.vim

```diff
@@ -113,6 +113,45 @@ function! Test_ResolveCompletion_OnChange()
   call test_override( 'ALL', 0 )
 endfunction
 
+function! Test_Resolve_FixIt()
+  call SkipIf( !exists( '*popup_findinfo' ), 'no popup_findinfo' )
+
+  " Only the java completer actually uses the completion resolve
+  call youcompleteme#test#setup#OpenFile(
+        \ '/third_party/ycmd/ycmd/tests/java/testdata/simple_eclipse_project' .
+        \ '/src/com/test/TestWithDocumentation.java', { 'delay': 15 } )
+
+  " Required to trigger TextChangedI
+  " https://github.com/vim/vim/issues/4665#event-2480928194
+  call test_override( 'char_avail', 1 )
+
+  function! Check1( id )
+    call WaitForCompletion()
+    call CheckCurrentLine( '    Tes' )
+    call CheckCompletionItemsHasItem( [ 'Test - com.youcompleteme' ] )
```

Comment:
Nitpicking, but shouldn't this be `CheckCOmpletionItemsHasItems` (trailing s), because you're looking for a subset, not just a single item?

Reply:
Good point. renamed.